### ISSUE TITLE
[RPC / Websocket] - Way to add CORS allowlist so we can allow all or specific website to directly connect.

### ISSUE THREAD

**Issue by maestroi** [nimiq/core-rs-albatross#2225]
## New issue checklist
- [X] I have read all of the [`README`](https://github.com/nimiq-network/core/blob/master/README.md) 
- [X] I have searched [existing issues](https://github.com/nimiq-network/core/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

### General information

- Library version(s): -
- Browser version(s): -
- Devices/Simulators/Machine affected: -
- Reproducible in the testnet? (Yes/No):  Yes
- Related issues: - 

Nimiq 0.20.0+

#### Expected behavior

I would expect to connect a browser directly to websocket/rpc interface with nimiq but thats not possible due to blocking in CORS, there is currently no way to add cors allowlist to all for example.

#### Actual behavior

Blocked by cors in browser.

#### Steps to reproduce

Connect via website to a rpc node directly.

#### Crash log? Screenshots? Videos? Sample project?

--

## Question or Feature Request

Add support in config to add CORS allowlist.
